lib/module.js

```diff
@@ -490,6 +490,10 @@ function tryModuleLoad(module, filename) {
   }
 }
 
+Module.import = function(request, parent) {
```

Comment:
Since this is only temporary until `import()` lands, even setting this property should be guarded by the `--experimental-modules` flag.